test/container_test.py

```diff
@@ -747,117 +882,67 @@ def test_webhook_setup_failure(servicer):
 
 
 @skip_github_non_linux
-def test_serialized_function(servicer):
-    def triple(x):
-        return 3 * x
-
-    ret = _run_container(
+def test_serialized_function(servicer, deployed_support_function_definitions):
+    ret = _run_container_auto(
         servicer,
-        "",  # no module name
-        "f",
-        definition_type=api_pb2.Function.DEFINITION_TYPE_SERIALIZED,
-        function_serialized=serialize(triple),
+        "serialized_triple",
+        deployed_support_function_definitions,
     )
     assert _unwrap_scalar(ret) == 3 * 42
 
 
 @skip_github_non_linux
-def test_serialized_class_with_parameters(servicer):
-    class SerializedClassWithParams:
-        p: int = modal.parameter()
-
-        @modal.method()
-        def method(self):
-            return "hello"
-
-        # TODO: expand this test to check that self.other_method.remote() can be called
-        #  this would require feeding the servicer with more information about the function
-        #  since it would re-bind parameters to the class service function etc.
-
-    app = modal.App()
-    app.cls(serialized=True)(SerializedClassWithParams)  # gets rid of warning
-
-    ret = _run_container(
+def test_serialized_class_with_parameters(servicer, deployed_support_function_definitions):
+    # TODO: expand this test to check that self.other_method.remote() can be called
+    #  this would require feeding the servicer with more information about the function
+    #  since it would re-bind parameters to the class service function etc.
+    ret = _run_container_auto(
         servicer,
-        "",
         "SerializedClassWithParams.*",
-        is_class=True,
+        deployed_support_function_definitions,
         inputs=_get_inputs(((), {}), method_name="method"),
-        definition_type=api_pb2.Function.DEFINITION_TYPE_SERIALIZED,
```

Comment:
This is the kind of huge duplicate definition stuff that we can omit when using `_run_container_auto` which instead relies on the already recorded definition (by the function decorator + deploy) instead of having devs add hand crafted container args.

This also prevents drift in definition between what the function deploy code (_load) does and what the test executes the container with